### ISSUE TITLE
[Accessiblity] Click target sizes

### ISSUE THREAD

**Issue by ErroneousBosch** [anyblades/blades#145]
By [WCAG Success Criterion 2.5.8](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html), non-inline click targets must be a minimum of 24x24px, which some elements do not meet. Notably:

* Default Semantic Details > Summary elements are normal line height (21px)
  * Proposal: add `padding: .1rem 0px;` and `margin: -.1rem 0px;`  css rules to take click target to 25.19px with minimal visual difference.
  * Button version does not have this issue
* Semantic Modal close button (Dialog > Article > Header > Button) is 1rem (18px)
  * Proposal: increase to `1.5rem` (27px) width and height.

You also have a small contrast issue with your menu dividers,  and some other site-specific issues with the Blades website, but these two were built into the CSS. I'd recommend doing some baseline A11y checking of your content.

If I find other accessibility issues, I will create issues.
